### PR TITLE
Added DKCR3D, Gunman Clive, Mutant Mudds, Pilotwings Resort, Snake Eater 3D, and Mighty Gunvolt in addition to adding more testcases/release info for SM 3D Land, SSB, and Link Between Worlds.

### PR DESCRIPTION
**Pull request by Anadian**
I tried my best to stay fairly close to the official spec (CONTRIBUTING.md) while adding some extra information of my own, such as, sha256 and comments.

### CODE REVIEW

**Comment by j-selby** [comment]
Comments are not needed, they should belong in the Wiki.

File: Super-Mario-Maker/game.dat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title = "Super Mario Maker"
+description = "Super Mario Maker is a side-scrolling platform video game and game creation system developed and published by Nintendo for the Wii U game console, which released worldwide in September 2015. An adapted port for the Nintendo 3DS, known as Super Mario Maker for Nintendo 3DS, was released in December 2016."
+needs_system_files = false
+needs_shared_font = true
+
+[[ releases ]]
+
```
Comment (j-selby) [inline Super-Mario-Maker/game.dat]: You are going to need something here.

File: Super-Mario-Maker/game.dat [hunk 2]
```diff
@@ -0,0 +1,8 @@
+title = "Super Mario Maker"
+description = "Super Mario Maker is a side-scrolling platform video game and game creation system developed and published by Nintendo for the Wii U game console, which released worldwide in September 2015. An adapted port for the Nintendo 3DS, known as Super Mario Maker for Nintendo 3DS, was released in December 2016."
+needs_system_files = false
+needs_shared_font = true
+
```
Comment (j-selby) [inline Super-Mario-Maker/game.dat]: This game suffers from startup issues, no?

File: Super-Mario-Maker/game.dat [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+[[ releases ]]
+
+[[ testcases ]]
```
Comment (j-selby) [inline Super-Mario-Maker/game.dat]: Ditto

File: Rhythm-Thief/game.dat [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+description = "Yoshi's Woolly World, known in Japan as Yoshi Wool World, is a side-scrolling platformer video game developed by Good-Feel and published by Nintendo for the Wii U. A port for the Nintendo 3DS, known as Poochy & Yoshi's Woolly World, was released worldwide in early 2017."
+needs_system_files = false
+needs_shared_font = true
+
+[[ releases ]]
+title = "0004000000067600"
+region = "USA"
+release_date = "2012-07-10"
+
+[[ testcases ]]
```
Comment (j-selby) [inline Rhythm-Thief/game.dat]: You need something here.

File: Poochy-Yoshis-Woolly-World/game.dat [hunk 5]
```diff
@@ -0,0 +1,8 @@
+title = "Poochy & Yoshi's Woolly World"
+description = "Yoshi's Woolly World, known in Japan as Yoshi Wool World, is a side-scrolling platformer video game developed by Good-Feel and published by Nintendo for the Wii U. A port for the Nintendo 3DS, known as Poochy & Yoshi's Woolly World, was released worldwide in early 2017."
+needs_system_files = false
+needs_shared_font = true
+
+[[ releases ]]
+
```
Comment (j-selby) [inline Poochy-Yoshis-Woolly-World/game.dat]: Ditto.

File: Poochy-Yoshis-Woolly-World/game.dat [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+[[ testcases ]]
```
Comment (j-selby) [inline Poochy-Yoshis-Woolly-World/game.dat]: Ditto.

**Comment by j-selby** [comment]
Take a look at the [Travis build log](https://travis-ci.org/citra-emu/citra-games-wiki/builds/244374526#L175) for additional errors.

**Comment by j-selby** [comment]
Could you also remove your sha checksums? Thanks - we don't have a use for them at this point.

**Comment by Anadian** [comment]
As @j-selby requested, I've removed the jpegs, comments and fixed all the syntax errors I'm aware of.

**Comment by j-selby** [comment]
You are still missing boxart for:
- Gunman Clive
- Mighty Gunvolt
- Mutant Mudds

You need at least one release and testcase in each game as well.

**Comment by j-selby** [comment]
Region names should also be lowercase (e.g. "aus")

**Comment by chris062689** [comment]
@Anadian Have a chance to update this yet? Thanks!

**Comment by Anadian** [comment]
I'm updating it right now.

**Comment by Anadian** [comment]
@chris062689 I've been busy the last few days but I'm updating it now.

**Comment by Anadian** [comment]
@j-selby Those are eshop games so, according to CONTRIBUTING.md, I'm supposed to take a screenshot of the title screen; what am I supposed to do when the title screen doesn't render correctly, or at all?

**Comment by Anadian** [comment]
I've removed everything that was out of compliance; it just seemed like the path of least resistance at this point.

**Comment by j-selby** [comment]
In terms of eShop titles, at least for a few of your titles fairly high-quality screenshots can be found on Google or other sources, as long as it reflects the original art.

I think you have got some of your compatibility scales mixed up. The lower the score, the better - you might want to review some of your scores in that regard.

Be warned this PR does now normalise line endings. While this isn't bad, it does do a global change.

**Comment by Anadian** [comment]
@j-selby  I'm aware that "[t]he lower the score, the better" policy for the compatibility score; where did you think I got it wrong?

**Comment by j-selby** [comment]
@Anadian I was looking at a previous version of your PR on that one - apologies.

**Comment by Anadian** [comment]
@j-selby No problem.

**Comment by Anadian** [comment]
Any reason this still hasn't been merged?

**Comment by chris062689** [comment]
Currently reviewing this. Should be merged soon.

**Comment by Anadian** [comment]
@chris062689 Thank you.

**Pull request merged**